lib/vsc/mympirun/mpi/intelmpi.py

```diff
@@ -176,16 +176,15 @@ def mpirun_prepare_execution(self):
     def pinning_override(self):
         """ pinning """
 
-        self.log.debug("pinning_override: type %s ", self.pinning_override_type)
+        logging.debug("pinning_override: type %s ", self.pinning_override_type)
 
         cmd = CmdList()
         if self.pinning_override_type in ('packed', 'compact', 'bunch'):
             cmd.add(['-env', 'I_MPI_PIN_PROCESSOR_LIST=allcores:map=bunch'])
         elif self.pinning_override_type in ('spread', 'scatter'):
             cmd.add(['-env', 'I_MPI_PIN_PROCESSOR_LIST=allcores:map=%s' % self.pinning_override_type])
         else:
-            self.log.raiseException("pinning_override: unsupported pinning_override_type  %s" %
-                                    self.pinning_override_type)
+            raise Exception("pinning_override: unsupported pinning_override_type  %s" % self.pinning_override_type)
```

Comment:
also log it, and removethe double space

Reply:
Any raised exception is logged by `main`, I'm deliberately not using `logging.error` here, or the same error message is duplicated, which is not very user friendly...

Reply:
i get that. the only risk is with caugth exceptions, but i guess the person who has to debug this will have to add more log statements